index.html

```diff
@@ -234,56 +234,54 @@ <h2 class="about_me_header">About<br>Myself</h2>
 						<p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
 						<div class="connect">
 							<div class="connect_text2">
-								<h2>LET'S CONNECT</h2>    
+								<h3>LET'S CONNECT</h3>    
 							</div>
 						</div>
 						<ul class="socials2">
 							<li>
 								<a href="https://twitter.com/GeorgeMsapenda" target="_blank">
-									<img src="images/Twitter icon.png" alt="Twitter link" class="hover_effect">
+									<img src="images/Twitter icon.png" alt="Twitter" class="hover_effect">
 								</a>
 							</li>
 							<li>
 								<a href="https://linkedin.com/in/george-m-sapenda-593750201" target="_blank">
-									<img src="images/Linkedin icon.png" alt="Linkedn link" class="hover_effect">
+									<img src="images/Linkedin icon.png" alt="Linkedn" class="hover_effect">
 								</a> 
 							</li>
 								<li>
 									<a href="https://github.com/c00p75" target="_blank">
-										<img src="images/Github icon.png" alt="Github link" class="hover_effect">
+										<img src="images/Github icon.png" alt="Github" class="hover_effect">
 									</a>
 							</li>                                
 						</ul>		
-						<div class="resume_link btn_effect">
-							<p>
-								<a href="#" id="resume">Get my resume</a>
-							</p>
+						<div class="resume_link btn_effect">							
+								<a href="#" id="resume">Get my resume</a>						
 						</div>		
 					</div>					
 					<ul class="skills">
 						<li class="skill_set1">
 							<div class="language_header">
-								<h2>Languages</h2>
+								<h3>Languages</h3>
 								<button class="btn2">
 									<img src="images/dropdown.png" alt="">
 								</button>
 							</div>
 							<ul class="language_list_container">
 								<li class="language">
 									<div class="lan">
-										<img src="images/js.png" alt="javascript icon">			
+										<img src="images/js.png" alt="javascript">			
 										<span class="language_name">JavaScript</span>
 									</div>
 								</li>
 								<li class="language">
 									<div class="lan">
-										<img src="images/html.png" alt="html icon">									
+										<img src="images/html.png" alt="html">									
 										<span class="language_name">HTML</span>
 									</div>
 								</li>				
 								<li class="language">
 									<div class="lan">
-										<img src="images/css.png" alt="css icon">
+										<img src="images/css.png" alt="css">
 										<span class="language_name">CSS</span>
 									</div>
 								</li>
```

Comment:
[OPTIONAL]
- [ ] Kindly consider fixing the size of all the buttons on your project cards

Your design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/200907309-a85f094d-10c5-4d3a-b92b-2249e6b2f7ad.png)

Figma design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/200907395-9f01f6e4-e728-45b2-a57a-6b795ea1f2d7.png)